backend/api/handlers/user/UpdatePasswordHandler.go

```diff
@@ -1,15 +1,75 @@
 package user
 
 import (
+	"errors"
 	"fmt"
-	"github.com/gin-gonic/gin"
+	"log"
 	"net/http"
+
+	"LiteracyLink.com/backend/api/model"
+	"LiteracyLink.com/backend/db"
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
 )
 
 func UpdatePasswordHandler(c *gin.Context) {
-	user_id := c.Param("user_id")
-	c.JSON(http.StatusAccepted, gin.H{
-		"status":  "success",
-		"message": fmt.Sprintf("TODO: make func let user with id: %s update password", user_id),
-	})
+	
+	var request model.PassUpdate
+	err := c.BindJSON(&request)
+	if err != nil {
+		log.Default()
+		log.Println(c.Request.Body)
```

Comment:
we probably should not log the request body because it can potentially contain a users password